home.html

```diff
@@ -26,17 +33,17 @@
   </head>
 
   <body>
-    <div class="container">
+    <div class="container-main">
       <nav class="menu-container show-menu" id="menu-list">
         <div class="img-container">
           <a href="#" class="hide-menu">
-            <img src="assets/times-solid.svg" width="27" height="27" alt=""
+            <img src="assets/times-solid.svg" alt=""
           /></a>
         </div>
         <div class="menu">
           <ul>
-            <li><a href="#portfolio"  class="hide-menu">Portfolio</a></li>
-            <li><a href="#aboutme"  class="hide-menu">About me</a></li>
+            <li><a href="#portfolio" class="hide-menu">Portfolio</a></li>
+            <li><a href="#aboutme" class="hide-menu">About me</a></li>
```

Comment:
- [ ] Kindly add the "X" logo when you click n the menu icon.
- [ ] Remember the menu icon should disappear when you click the "X" logo.
- [ ] When I open the menu in the mobile layout, then I will change the screen size, the menu from the mobile layout is still there, please, fix it. You can see how there are the menu nav links from the desktop version, but also you still have the menu from the mobile version.

Sample:

![image](https://user-images.githubusercontent.com/60164239/131576505-2f051b0e-74b4-419d-9696-b94626442e2c.png)


- [ ] [OPTIONAL] As you can see in the background the Figma design is blurred, it would be nice if you add this effect.
- [ ] [OPTIONAL] It would be nice if you add the right blue color, you can check out in the Figma design what exactly is this color 😉 

Your design | Figma design

![image](https://user-images.githubusercontent.com/60164239/131575795-2aa9068f-41e0-4365-ab3a-0fb9ac81bdf1.png)